index.js

```diff
@@ -389,14 +388,13 @@ function includeFlashPlugin() {
 }
 
 function initializeCrashReporter(argo) {
-    if (!crashReporterEnabled && argo['enable-crash-reporting']) {
-        crashReporter.start({
-            productName: 'OpenFin',
-            companyName: 'OpenFin',
-            submitURL: 'https://dl.openfin.co/services/crash-report',
-            autoSubmit: true
-        });
-        crashReporterEnabled = true;
+    const enableCrashReporting = argo['enable-crash-reporting'];
+    const diagnosticMode = argo['diagnostics'] || false;
+    const configUrl = argo['startup-url'];
```

Comment:
@whyn07m3 I'll update, but if you want this for real in the future lets somehow make it a lint rule, not something we catch in review